app/scripts/lib/experiment.js

```diff
@@ -114,7 +114,7 @@ define(function (require, exports, module) {
         return;
       }
 
-      for (let experimentName in this._allExperiments) {
+      for (const experimentName in this._allExperiments) {
```

Comment:
@shane-tomlinson, CMIIW, but I think that is preferable for the traditional for loop, like so:
`for(let i = 0; i < 5; i++)`
Using const should work fine with the for-in loop and the for-of loop.